.gitignore

```diff
@@ -8,4 +8,7 @@
 /.vscode
 
 # Build artifact that must be placed alongside go files for Windows
-*.syso
\ No newline at end of file
+*.syso
+
+# JetBrains
+.idea
```

Comment:
Just a heads up; you can add this to `~/.gitignore`

```
/.idea
```

And in `~/.gitconfig`
```
[core]
	excludesfile = /Users/dom/.gitignore
```

Then it will be automatically ignored in all your repo's

Reply:
✅